grammars/jsdoc.cson

```diff
@@ -410,6 +410,10 @@
           }
         ]
       }
+      {
+        'match': '^\\s*\\*(?!/)'
+        'name': 'comment.block.documentation.jsdoc'
```

Comment:
We occasionally get spammers due to the popularity of the Atom project.  I think this comment is safe to ignore.